recipes/pylogit/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "pylogit" %}
+{% set version = "0.2.2" %}
+{% set sha256 = "a6809e2adb3063fe1408696080217440ab6ea5eb1f3aab62eba9e215559e2fbc" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
```

Comment:
It looks like there are some version restrictions on the dependencies:

https://github.com/timothyb0912/pylogit/blob/master/setup.py#L88-L93

Reply:
I have added these.